### PR TITLE
feat(timepicker): add accessibility improvements

### PR DESCRIPTION
**Pull request by wesleycho**
- Add ng-disabled usage in template
- Add tabindex to support to timepicker controls

This addresses #4569

### CODE REVIEW

File: src/timepicker/docs/readme.md [hunk 1]
```diff
@@ -50,3 +50,7 @@ All settings can be provided as attributes in the `<uib-timepicker>` or globally
  * `max`
     _(Defaults: undefined)_ :
      Maximum time a user can select
+
+ * `timepicker-tabindex`
```
Comment (pkozlowski-opensource) [inline src/timepicker/docs/readme.md]: couldn't we use `tabindex` here? And remove it from the timepicker host element if needed?

Comment (wesleycho) [inline src/timepicker/docs/readme.md]: I thought about it, but I had a concern...but I can't remember it now - I am willing to go that route though for simplicity.

**Comment by Foxandxss** [comment]
Not really my topic.

The only bit to talk here is that we shouldn't be supporting the deprecated directives. The point is to force people to upgrade and not fixing them is a good way to do so.

**Comment by wesleycho** [comment]
This has nothing to do with deprecated directive support :) .

**Comment by Foxandxss** [comment]
I know I know, I just said that tabs and accessibility is not something I can talk about.

**Pull request closed without merging**